yadm

```diff
@@ -169,7 +169,7 @@ function alt() {
 
   #; decide if a copy should be done instead of a symbolic link
   local do_copy=0
-  if [[ $OPERATING_SYSTEM == CYGWIN* ]] ; then
+  if [[ $OPERATING_SYSTEM =~ ^(CYGWIN|MSYS) ]] ; then
```

Comment:
@rasa I was just merging this into the develop branch, and I noticed something different between your change and your comments in this PR.

You're mention that `uname -o` will be either "Cygwin" or "Msys" in these cases, but when testing the $OPERATING_SYSTEM, you're testing agains the regex `^(CYGWIN|MSYS)`.

Can you see if these should be initial caps or all caps? I do not have a Windows system to test with. My guess is that this should be `^(Cygwin|Msys)` based on the other information, including Wikipedia.

Maybe this doesn't matter too much, as you've suggested universal supporting an option to for alternate "copies". I think I may do just that.

Reply:
You are correct:
```
ross@indra ~
$ uname -o
Cygwin
ross@indra ~
$ uname -s
CYGWIN_NT-10.0
```
```
ross@indra:~$ uname -o
Msys
ross@indra:~$ uname -s
MSYS_NT-10.0
```

The commit https://github.com/TheLocehiliosan/yadm/pull/102/commits/54f7cbcebe835b098a23998a16eb11f4d643eeb1 is wrong. https://github.com/TheLocehiliosan/yadm/pull/102/commits/315ad0873e843b1cfa7f2a9933e15dde798cba24 is correct. Sorry for the mixup.

Reply:
No worries. Your comments were enough for me to realize it was probably like that.

I did actually strip out the OS restriction on that feature (changing the option to `yadm.alt-copy`). But knowing the actual values for OPERATING_SYSTEM will be helpful for documentation examples, etc.  Thanks.